jvb/src/main/java/org/jitsi/videobridge/stats/VideobridgeStatistics.java

```diff
@@ -116,12 +119,16 @@
     private final @NotNull Videobridge videobridge;
     private final @NotNull XmppConnection xmppConnection;
 
+    private final BooleanMetric healthy = MetricsContainer.getInstance().registerBooleanMetric(
```

Comment:
I think it would be nice to have an optional initial value, so we can have this start off as `true`.

Reply:
Agreed!